backend/src/main/java/de/wirvsvirus/kek/service/diary/model/ContactMethod.java

```diff
@@ -0,0 +1,73 @@
+package de.wirvsvirus.kek.service.diary.model;
```

Comment:
I think we should rename ContactMethod. Because It might cause confusion with contactdiary/contactentry...
CorrespondenceDetails maybe?

The rest LGTM.